covquestions-js/src/questionnaireEngine.ts

```diff
@@ -281,8 +319,41 @@ export class QuestionnaireEngine {
     });
     return results.filter(notUndefined);
   }
+  public getResults(): QuestionnaireResult {
+    return {
+      answers: this.givenAnswers.reduce((aggregate, current) => {
+        aggregate[current.questionId] = current.rawAnswer;
+        return aggregate;
+      }, {} as { [key: string]: RawAnswer }),
+      exports: [
```

Comment:
Do we want to do the filtering for "qr" in here or in the consuming CovApp / the XML mapping?

I suggest to add just the variables here (next to answers and results): variables: { [key: string]: string }

Reply:
I just wanted to implement the final interface for the exports already. 
The QR code will be mapped via the `id` and all including keys

Reply:
also, I thought about splitting the getResults function into results (as previously) and new `getExports` function and rewriting the persistence functions, because we are kind of duplicating functionality.

Reply:
Alright. Sounds reasonable.